### PR TITLE
add Mattermost (chat)

### PR DESCRIPTION
**Pull request by veggiemonk**

### CODE REVIEW

**Comment by nodiscc** [comment]
Thank you for the patch.

Mattermost should be in the [Custom](https://github.com/Kickball/awesome-selfhosted#custom) section, not xmpp.

License and prog language should be written as `AGPLv3/Apache 2.0` `Go/Javascript`

Can you do the required changes?

**Comment by veggiemonk** [comment]
@nodiscc There you go. :smile:

**Comment by veggiemonk** [comment]
@nodiscc Thanks for fixing it.

File: README.md [hunk 1]
```diff
@@ -778,6 +779,7 @@ See also [Documentation Generators](#documentation-generators), [Wikimatrix](htt
  * Non-[Free](https://en.wikipedia.org/wiki/Free_software) software must be marked `⊘ Proprietary` next to the title, and added to [non-free.md](non-free.md)
  * Software with no development activity for 6-12 months may be removed from the list.
  * The syntax for links is ``[Name](http://homepage/) - Short description ([Demo](http://url.to/demo), [Source code](http://url.of/source/code)) `License` `Language` ``
+ * License and programming language should be written as AGPLv3/Apache 2.0 Go/Javascript
```
Comment (nodiscc) [inline README.md]: Please remove this line

**Pull request merged**